composer.json

```diff
@@ -84,6 +84,7 @@
         "drupal/views_data_export": "^1.3",
         "drupal/views_field_view": "^1.0",
         "drupal/ultimate_cron": "^2.0@alpha",
+        "eluceo/ical": "^0.17",
```

Comment:
I've talked with some guys and they suggested using the 0.17 version because not all features are working with a new version right now.
Some contrib modules use 0.17 version as well. e.g. views_ical module.